dashboard/package.json

```diff
@@ -29,14 +30,21 @@
     "lodash": "^4.17.21",
     "next": "^12.0.9",
     "react": "^18.2.0",
+    "react-bootstrap": "1.6.0",
```

Comment:
We're using https://chakra-ui.com as our UI framework, so I don't think it's necessary to introduce react-bootstrap.